addons/payment_razorpay/models/payment_transaction.py

```diff
@@ -54,7 +54,7 @@ def _razorpay_create_customer(self):
         """
         payload = {
             'name': self.partner_name,
-            'email': self.partner_email,
+            'email': self.partner_email or "",
```

Comment:
Same style as the rest.
```suggestion
            'email': self.partner_email or '',
```